Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>"if first_shell_index == None" - PEP8 best practice is to use "if first_shell_index is None"</p><p>Guidnace is that comparisons to singletons like None should always be done with is or is not, never the equality operators.</p><p> </p><p>Python3 added f-strings which are a more robust way of doing string formatting (they can handle things like the substitute value not being a string).</p><p>Suggest you use print(f"No files were access by the attacker prior to webshell install, try expaning the query window (currently: '{queryWindow}')") and similar</p><p>f-string overview - https://realpython.com/python-f-strings/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thanks for spotting Pete, I was using f-strings elsewhere after discovering it in some of your notebook, I have migrated to this in the final cells I had previously overlooked.</p><p> </p><p>Made my singleton comparisons complaint with PEP8, appreciate you highlighting this style guide, it's new to me.</p>